src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -310,6 +310,10 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 			this._startBlockingIframeDragEvents();
 		}));
 
+		this._register(this.on('drag', (event) => {
+			this.handleDragEvent('drag', event);
```

Comment:
we likely need to prevent default _after_ we pass this to our custom drag event listener